main.go

```diff
@@ -186,6 +187,7 @@ func configureParameters(sourcesWhiteList []string, labelWhiteListStr string) (e
 	// Configure feature sources.
 	allSources := []source.FeatureSource{
 		cpuid.Source{},
+		iommu.Source{},
```

Comment:
here is practically same question: is the order of the elements matter ? if not, should it be added in the end then ?

Reply:
Same here